sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/WebSocketsConnectionHandler.java

```diff
@@ -24,16 +24,26 @@ public class WebSocketsConnectionHandler extends ConnectionHandler {
 
     private static final String SOCKET_PATH = "/$servicebus/websocket";
     private static final String PROTOCOL = "AMQPWSB10";
+    /**
+     * Once there is an HTTP Connection to the host addressable by https://host-name
+     * (connection the client 'directly' established or established by tunneling through
+     * Proxy etc..), the WebSocket layer has to send an Upgrade request (GET https://host-name)
+     * with upgrade-specific headers to switch from HTTP to WebSocket protocol.
+     * The host-name is the FQDN of the Event Hubs or Service Bus or host part of
+     * CustomEndpointAddress when a custom endpoint frontends the Event Hubs or Service Bus.
+     * The upgrade request will have a 'Host' header matching the host-name.
+     */
+    private final String hostName;
```

Comment:
for consistency: `hostname` rather than `hostName`

Reply:
Agree, done.